OneOneOne/OneOneOne/tests/test_message_handler.py

```diff
@@ -0,0 +1,154 @@
+import unittest
+from pathlib import Path
+from builder.pystache_message_builder import PystacheMessageBuilder
+from utilities.file_utilities import FileUtilities
+from utilities.xml_utilities import XmlUtilities
+from OneOneOne.OneOneOne.message_handler import MessageHandler
+
+from definitions import ROOT_DIR
+
+
+class MessageHandlerTest(unittest.TestCase):
+    expectedXmlFileDir = Path(ROOT_DIR) / 'OneOneOne' / 'tests' / 'expected_output_xmls'
```

Comment:
Given your expected output XMLs are so similar, would it not make sense to produce the expected output from a mustache template?

Reply:
I can see why this would be better, but also feels a bit strange as I'd effectively be running the same code twice and asserting their outputs would be the same, which just seems like repeating the task rather than asserting it against what we know is correct?

Reply:
A fair point. I'm happy to leave the expected output XMLs as they are then.